app/src/marketplace/containers/ProjectEditing/DeleteProject.tsx

```diff
@@ -29,6 +33,11 @@ const DeleteProject = () => {
                 onClick={async () => {
                     try {
                         await deleteProject(project?.id || undefined)
+
+                        // Navigate away from now gone project (if user stayed on the edit page)
+                        if (pathname === routes.projects.edit({ id: project?.id })) {
+                            history.push(routes.projects.index())
+                        }
```

Comment:
That's cool. Then if the user goes back to the edit page they'll be taken away by the site.